home:
  position: 
    x: 2.0
    y: 0.0
    z: 0.0
  orientation: 
    x: 0.0
    y: 0.0
    z: 0.0
    w: 1.0

location:
  position: 
    x: 0.0
    y: 0.0
    z: 0.0
  orientation: 
    x: 0.0
    y: 0.0
    z: -1.0
    w: 0.0

chairs:
  # middle
  chair0:
    status: 'unknown'
    box:
      frame: 'map'
      max_xy: [ -2, 0 ]
      min_xy: [ -3,-1 ]
    neighbours: [chair1, chair2]
  # right
  chair1:
    status: 'unknown'
    box:
      frame: 'map'
      max_xy: [ -2,-1 ]
      min_xy: [ -3,-2 ]
    neighbours: [chair0]
  # left
  chair2:
    status: 'unknown'
    box:
      frame: 'map'
      max_xy: [ -2, 1.5 ]
      min_xy: [ -3, 0 ]
    neighbours: [chair0]